examples/plugins/rotator.js

```diff
@@ -0,0 +1,20 @@
+module.exports = bot => {
+  let rotater
+  let rotated = false
+  bot.afk = {}

```

Comment:
The name should still be afk imo
+ Can you add back the comments ?

Reply:
done